wpilibc/athena/src/RobotDrive.cpp

```diff
@@ -625,14 +610,15 @@ void RobotDrive::SetLeftRightMotorOutputs(double leftOutput,
 /**
  * Limit motor values to the -1.0 to +1.0 range.
  */
-double RobotDrive::Limit(double num) {
-  if (num > 1.0) {
-    return 1.0;
-  }
-  if (num < -1.0) {
-    return -1.0;
-  }
-  return num;
+double RobotDrive::Limit(double number) {
```

Comment:
You could add a header to define it like we did with std::make_unique if you want. There's example "possible implementations" at http://en.cppreference.com/w/cpp/algorithm/clamp